sdk/cosmos/azure-cosmos-test/src/samples/java/com/azure/cosmos/test/ReadmeSamples.java

```diff
@@ -122,4 +127,153 @@ public void randomConnectionCloseScenario() {
         CosmosFaultInjectionHelper.configureFaultInjectionRules(container, Arrays.asList(connectionErrorRule)).block();
         // END: readme-sample-randomConnectionCloseScenario
     }
+
+    /**
+     * Demonstrates fault injection for response delay using GATEWAY V2 and HTTP/2 configuration.
+     */
+    public void responseDelayWithGatewayV2Scenario() {
+        // BEGIN: readme-sample-responseDelayWithGatewayV2Scenario
+        // Enable thin client and configure HTTP/2
+        System.setProperty("COSMOS.THINCLIENT_ENABLED", "true");
```

Comment:
I think probably also need to `System.setProperty("COSMOS.HTTP2_ENABLED", "true")` or `new Http2ConnectionConfig().setEnabled(true)`

Reply:
Fixed